src/lsm/groove.zig

```diff
@@ -419,11 +419,10 @@ pub fn GrooveType(
         const trees_total = @as(usize, 1) + @intFromBool(has_id) + std.meta.fields(IndexTrees).len;
         const TreesBitSet = std.StaticBitSet(trees_total);
 
-        const PrefetchKey = struct { key: union(enum) {
+        const PrefetchKeys = std.AutoHashMapUnmanaged(union(enum) {
             id: PrimaryKey,
             timestamp: u64,
-        }, level: u8 };
-        const PrefetchKeys = std.AutoHashMapUnmanaged(PrefetchKey, void);
+        }, u8);
```

Comment:
Nit, but I think I would change the `u8` to be a single-field struct: `struct { level: u8 }` to make it more obvious what the `u8` represents.